### PR TITLE
chore: design unification

### PR DESCRIPTION
**Pull request by jzempel**
## Description

This PR represents a sweeping cleanup task intended to bring the site inline with corresponding designs. Along with many typography and spacing tweaks, notable items include:

- sourcing homepage images from Abstract
- copyright stripped from "show code" blocks to reduce visual noise (the code is already covered under the site's ©️ ); remains intact for copy/paste and codesandbox
- code examples incorporating `ToggleIconButton` actions for "RTL" and "Show code"
- Garden's standard `.prettierrc` included for codesandbox forks
- API table re-styled to horizontally scroll on mobile

## Detail

Please note that home page graphics have changed. I'll leave it to @rossmoody to rationalize the difference between this PR and the current https://zendeskgarden.netlify.app/.

## Checklist

- [x] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: ~copy updates are approved (add the content strategist as a reviewer)~
- [x] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [x] :zap: audited via [web.dev](https://web.dev/measure/) for performance and SEO
- [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/components/MarkdownProvider/components/BestPractice.tsx [hunk 1]
```diff
@@ -44,6 +45,8 @@ const StyledImg = styled.img`
   border-bottom: none;
   border-top-left-radius: ${p => p.theme.borderRadii.md};
   border-top-right-radius: ${p => p.theme.borderRadii.md};
+  padding: ${p => p.theme.space.md};
+  object-fit: cover;
```
Comment (hzhu) [inline src/components/MarkdownProvider/components/BestPractice.tsx:49]: I noticed that `object-fit` is listed as unsupported for IE11. Does the image fit okay in IE11? 

https://caniuse.com/#feat=object-fit

Comment (jzempel) [reply src/components/MarkdownProvider/components/BestPractice.tsx:49]: Similar to https://github.com/zendeskgarden/react-components/blob/5bebfa80a1155452eed1aae61f2041f9efbd4691/packages/avatars/src/styled/StyledAvatar.ts#L262, we want to optimize for the browsers that can benefit. In general, image skew will be addressed when the designers resize taking padding into account.

File: src/components/MarkdownProvider/components/CodeExample/components/SyntaxHighlighter.tsx [hunk 2]
```diff
@@ -7,10 +7,10 @@
 
 import React from 'react';
 import ReactSyntaxHighlighter from 'react-syntax-highlighter';
-import dracula from 'react-syntax-highlighter/dist/esm/styles/hljs/dracula';
+import style from 'react-syntax-highlighter/dist/esm/styles/hljs/atom-one-dark';
```
Comment (austin94) [inline src/components/MarkdownProvider/components/CodeExample/components/SyntaxHighlighter.tsx:10]: [nit] is it useful to name the import `atomOneDarkStyle` or something more descriptive?

File: src/components/MarkdownProvider/components/CodeExample/index.tsx [hunk 3]
```diff
@@ -76,45 +71,67 @@ export const CodeExample: React.FC<{ code: string }> = ({ children, code }) => {
...
+              `}
+            >
               <LightningBoltStroke />
-            </StyledFooterButton>
+            </IconButton>
           </Tooltip>
         </form>
         <Tooltip content={tooltipContent}>
-          <StyledFooterButton
-            aria-label="Copy to clipboard"
```
Comment (austin94) [inline src/components/MarkdownProvider/components/CodeExample/index.tsx:99]: Was this `aria-label` removal intended?

File: src/components/MarkdownProvider/components/CodeExample/index.tsx [hunk 4]
```diff
@@ -76,45 +71,67 @@ export const CodeExample: React.FC<{ code: string }> = ({ children, code }) => {
...
+              margin-left: ${p => p.theme.space.sm};
+            `}
           >
             <CopyStroke />
-          </StyledFooterButton>
+          </IconButton>
         </Tooltip>
         <Tooltip content={isCodeVisible ? 'Hide code' : 'Show code'}>
-          <StyledFooterButton
-            aria-label={isCodeVisible ? 'Hide code' : 'Show code'}
```
Comment (austin94) [inline src/components/MarkdownProvider/components/CodeExample/index.tsx:110]: Was this `aria-label` removal intended?

File: src/components/MarkdownProvider/components/CodeExample/components/SyntaxHighlighter.tsx (hunk 2 above)
Comment (jzempel) [reply src/components/MarkdownProvider/components/CodeExample/components/SyntaxHighlighter.tsx:10]: We're playing with the different options over time. I didn't want to keep chasing down the variable name.

File: src/components/MarkdownProvider/components/CodeExample/index.tsx (hunk 3 above)
Comment (jzempel) [reply src/components/MarkdownProvider/components/CodeExample/index.tsx:99]: Yes. The tooltip imparts `aria-describedby` to the button making `aria-label` redundant.

File: src/components/MarkdownProvider/components/CodeExample/index.tsx (hunk 4 above)
Comment (jzempel) [reply src/components/MarkdownProvider/components/CodeExample/index.tsx:110]: Yes, due to the Tooltip.

File: src/components/MarkdownProvider/components/BestPractice.tsx (hunk 1 above)
Comment (jzempel) [reply src/components/MarkdownProvider/components/BestPractice.tsx:49]: ...but scaling on IE11 can definitely be improved. Fixing that now.

File: src/components/MarkdownProvider/components/CodeExample/index.tsx [hunk 5]
```diff
@@ -24,20 +22,17 @@ const StyledExampleWrapper = styled.div`
...
-  margin-left: ${p => p.theme.space.xxs};
-`;
-
 export const CodeExample: React.FC<{ code: string }> = ({ children, code }) => {
   const [isRtl, setIsRtl] = useState(false);
   const [isCodeVisible, setIsCodeVisible] = useState(false);
   const focusVisibleRef = useRef(null);
   const initialTooltipContent = 'Copy to clipboard';
   const [tooltipContent, setTooltipContent] = useState(initialTooltipContent);
+  const COPYRIGHT_REGEXP = /\/\*\*\n\s\*\sCopyright[\s\S]*\*\/\n\n/gmu;
```
Comment (rossmoody) [inline src/components/MarkdownProvider/components/CodeExample/index.tsx:35]: 🎉

**Pull request merged**